internal/yggdrasil/yggdrasil.go

```diff
@@ -508,6 +515,33 @@ func (h *Handler) toWorkloadList(ctx context.Context, logger logr.Logger, deploy
 	return list, nil
 }
 
+// validatePodmanLabels validate the podman labels
+func validatePodmanLabels(podmanLabels map[string]string, deployment v1alpha1.EdgeDeployment) bool {
+	for key, value := range podmanLabels {
+		if strings.Contains(key, "io.containers.autoupdate.authfile") {
+			for _, ctr := range deployment.Spec.Pod.Spec.Containers {
+				for _, vm := range ctr.VolumeMounts {
+					if value == vm.MountPath {
+						return true
+					}
+				}
+			}
+		}
+	}
+	return false
+}
+
+// getPodmanLabels filter all the labels of the EdgeDeployment CR starting with prefix "podman/"
+func getPodmanLabels(deploymentLabels map[string]string) map[string]string {
+	labels := map[string]string{}
+	for key, value := range deploymentLabels {
+		if strings.HasPrefix(key, "podman/") {
```

Comment:
just an idea, since we have flottalabels.go, do you think it should go into there as well?
probably can be useful for reusing it from the unit-tests.